src/NUnitTestExtractor/NUnitTestExtractor/Program.cs

```diff
@@ -32,68 +33,132 @@ static void Main(string[] args)
                    
                     if(_options.Output != null)
                     {
-                        if (!Directory.Exists(_options.Output))
+                        if (Path.GetExtension(_options.Output) != ".txt")
+                        {
```

Comment:
I don't think we should restrict what the filename should be.  The user can call the file foo.bar for all I care.  :)
Some people might use a .csv extension...